Signal extension doc updates

spoorthipujariadobe opened the pull request.

calebk1 commented on foundation-extensions/mobile-core/signals/README.md:
```diff
@@ -1,18 +1,18 @@
 # Signal
 
-The Signal extension allows marketers to send a "signal" to their apps through the Experience Platform SDKs. This signal might tell the SDKs or the apps to complete tasks, such as send PII-labeled data, to trigger a postback to a third-party ad-network and open an app deep link or URL. To ensure that signals are sent or are activated, the marketers need to configure triggers and traits in Experie ...
+The Signal extension allows marketers to send a "signal" to their apps through the Adobe Experience Platform Mobile SDKs. This signal might tell the Mobile SDKs or the apps to complete tasks, such as send PII-labeled data, to trigger a postback to a third-party ad-network and open an app deep link or URL. To ensure that signals are sent or are activated, the marketers need to configure triggers a ...
```
"... traits in the Data Collection UI."

calebk1 commented on foundation-extensions/mobile-core/signals/README.md:
```diff
@@ -1,18 +1,18 @@
+The Signal extension allows marketers to send a "signal" to their apps through the Adobe Experience Platform Mobile SDKs. This signal might tell the Mobile SDKs or the apps to complete tasks, such as send PII-labeled data, to trigger a postback to a third-party ad-network and open an app deep link or URL. To ensure that signals are sent or are activated, the marketers need to configure triggers a ...
 
-The Signal extension is bundled with the [MobileCore \(Android\)/ACPCore \(iOS\)](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/) extension and allows you to send postbacks to third-party endpoints and open URLs, such as web URLs or application deep links, when using rules actions in Experience Platform Launch.
+The Signal extension is bundled with the [MobileCore (Android)/ACPCore (iOS)](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/) extension and allows you to send postbacks to third-party endpoints and open URLs, such as web URLs or application deep links, when using rules actions in Data Collection UI.
```
"... in the Data Collection UI."

calebk1 commented on foundation-extensions/mobile-core/signals/README.md:
```diff
@@ -1,18 +1,18 @@
 1. Add the **Signal** extension to your app.
-2. Define the necessary rules in Experience Platform Launch. 
-3. \(Optional\) When using Send PII actions in Experience Platform Launch, implement the APIs to collect PII data and send it to the configured third party destination.
+2. Define the necessary rules in Data Collection UI. 
```
"... in the Data Collection UI."

calebk1 commented on foundation-extensions/mobile-core/signals/README.md:
```diff
@@ -1,18 +1,18 @@
-2. Define the necessary rules in Experience Platform Launch. 
-3. \(Optional\) When using Send PII actions in Experience Platform Launch, implement the APIs to collect PII data and send it to the configured third party destination.
+2. Define the necessary rules in Data Collection UI. 
+3. (Optional) When using Send PII actions in Data Collection UI, implement the APIs to collect PII data and send it to the configured third party destination.
```
"... in the Data Collection UI, implement..."

calebk1 commented on foundation-extensions/mobile-core/signals/README.md:
```diff
@@ -1,18 +1,18 @@
+3. (Optional) When using Send PII actions in Data Collection UI, implement the APIs to collect PII data and send it to the configured third party destination.
 
-For more information about creating and configuring a rule in Experience Platform Launch, see [Rules](https://experienceleague.adobe.com/docs/experience-platform/tags/ui/rules.html?lang=en).
+For more information about creating and configuring a rule in Data Collection UI, see [Rules](https://experienceleague.adobe.com/docs/experience-platform/tags/ui/rules.html?lang=en).
```
"... in the Data Collection UI, see the [guide on rules]."

calebk1 commented on foundation-extensions/mobile-core/signals/README.md:
````diff
@@ -173,7 +174,7 @@ public class MobileApp extends Application {
 ```
 
-**Important**: The Signal extension is automatically included in Core by Maven. When you manually install the Signal extension, ensure that you add the `signal-1.x.x.aar` library to your project.
+**Important**: The Signal extension is automatically included in Mobile Core extension by Maven. When you manually install the Signal extension, ensure that you add the `signal-1.x.x.aar` library to your project.
````
"... in the Mobile Core extension..."

calebk1 commented on foundation-extensions/mobile-core/signals/README.md:
```diff
@@ -269,28 +273,28 @@ void Start()
 private void startCallback()
 {
-  // set launch config
+  // set app id from Data Collection UI
```
"Set app ID from the Data Collection UI"

calebk1 commented on foundation-extensions/mobile-core/signals/README.md:
```diff
@@ -299,15 +303,15 @@ protected override void OnCreate(Bundle savedInstanceState)
   public void Call(Java.Lang.Object callback)
   {
-    // set launch config
+  // set app id from Data Collection UI
```
"set app ID from the Data Collection UI"

calebk1 commented on foundation-extensions/mobile-core/signals/README.md:
```diff
@@ -321,5 +325,5 @@ To send PII data to external destinations, the `PII` action can trigger the Rule
 For more information about `collectPii` and its usage, see [collectPii](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii).
 
-For more information about how to configure the Signal postbacks in Experience Platform Launch, see [Signal extension and Rules Engine integration](https://aep-sdks.gitbook.io/docs/resources/user-guides/signal-extension-and-rules-engine-integration).
+For more information about how to configure the Signal postbacks in Data Collection UI, see [Signal extension and Rules Engine integration](https://aep-sdks.gitbook.io/docs/resources/user-guides/signal-extension-and-rules-engine-integration).
```
"... in the Data Collection UI, see the guide on the [Signal extension and Rules Engine integration]."

calebk1 reviewed (approved):
Looks good!

The major thing is just to change it from stuff like "... traits in Data Collection UI" to "... traits in the Data Collection UI." (since it needs a the in front).

But other than that, the changes are good!

spoorthipujariadobe commented:
> Looks good!
> 
> The major thing is just to change it from stuff like "... traits in Data Collection UI" to "... traits in the Data Collection UI." (since it needs a the in front).
> 
> But other than that, the changes are good!

Addressed review comments, please feel free to merge if no more changes are need.

emdobrin commented on foundation-extensions/mobile-core/signals/README.md:
````diff
@@ -42,23 +42,23 @@ import com.adobe.marketing.mobile.*;
-pod 'AEPCore'
-pod 'AEPSignal'
+```pod
+pod 'AEPCore','~>3.0'
+pod 'AEPSignal','~>3.0'
````
Can you include a space before the version: `~> 3.0`

emdobrin commented on foundation-extensions/mobile-core/signals/signal-api-reference.md:
```diff
@@ -10,37 +10,62 @@ To get the version of the Signal extension, use the following code sample:
+
 {% endtab %}
 
 {% tab title="React Native" %}
```
Let's also use relative link for the collectPII link below.

emdobrin replied:
Looks like this one got renamed to `../mobile-core-api-reference.md#collectpii`

emdobrin commented on foundation-extensions/mobile-core/signals/signal-api-reference.md:
```diff
@@ -10,37 +10,62 @@ To get the version of the Signal extension, use the following code sample:
 {% tab title="Android" %}
```
Rename header `Version of the Signal extension` -> `extensionVersion`

emdobrin commented on foundation-extensions/mobile-core/signals/signal-api-reference.md:
````diff
@@ -10,37 +10,62 @@ To get the version of the Signal extension, use the following code sample:
+```objectivec
+NSString *signalExtensionVersion = [ACPSignal extensionVersion];
+```
+
````
Can we include the registerExtension API too? Example https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#registerextension

emdobrin commented on foundation-extensions/mobile-core/signals/signal-api-reference.md:
````diff
@@ -113,7 +113,57 @@ string signalVersion = ACPSignal.ExtensionVersion();
+```
+{% endtab %}
+
+{% tab title="iOS \(AEP 3.x\)" %}
````
Can you also remove back-slashes here and for 2.x?

emdobrin reviewed (approved):
Looks good with couple small comments
